### PR TITLE
Feature/pp9 8518 create share dialog

### PR DESCRIPTION
**Pull request by tapmapp**
This is the implementation of the share dialog for the selected images.

### CODE REVIEW

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/components/share-images-dialog/share-images-dialog.component.html [hunk 1]
```diff
@@ -4,9 +4,18 @@ <h3>Create Share <mat-icon (click)="closeDialog()">close</mat-icon></h3>
 
   <mat-dialog-content>
     <ul>
-      <li><label for="share-name">Share name:</label><input id="share-name" type="text"></li>
-      <li><label for="recipients">Recipients:</label><input id="recipients" type="text"></li>
-      <li><label for="expiry-date">Expiry date:</label><pp-date-picker [placeHolderMessage]="'Expiration Date'"></pp-date-picker></li>
+      <li class="label-content"><label for="share-name">Share name:</label></li>
+      <li class="input-content">
+        <input id="recipients" type="text">
```
Comment (tapmapp) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/components/share-images-dialog/share-images-dialog.component.html]: should be '<input id="share-name"...'

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/components/chip-input/chip-input.component.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    if (input) {
+      input.value = '';
+    }
+  }
+
+  remove(fruit: Email): void {
+    const index = this.fruits.indexOf(fruit);
+
+    if (index >= 0) {
+      this.fruits.splice(index, 1);
```
Comment (tapmapp) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/components/chip-input/chip-input.component.ts]: change 'fruits' for 'selectedImages'

**Comment by neotek** [comment]
Add share button to basket.

**Comment by neotek** [comment]
Fix Cannot read property 'nativeElement' of undefined

**Comment by neotek** [comment]
Prefill share name.

**Pull request merged**